### DISCUSSION TITLE
[Feature]

### DISCUSSION THREAD

**Discussion by mawsyh** [klinecharts/KLineChart#413] [Ideas]
### Feature Description

Do we already have a feature to disable MouseMove event on Y and X axis ?
I didn't see anything on doc or code.

**Comment by liihuu** [comment]
Do you have any good suggestions for using APIs or configuring them?

**Reply by mawsyh** [reply]
working on it

**Comment by liihuu** [comment]
#398

**Comment by mawsyh** [comment]
I added a new option called yScrolling , which give you the ability to disable or enable scrolling vertically from first (as my employer wanted this option). I also change kline pro wrapper to work properly with this new option. (I forgot to add it to docs tho if you merge the changes I will change the docs as well)
https://github.com/liihuu/KLineChart/pull/414
https://github.com/klinecharts/pro/pull/7

**Reply by liihuu** [reply]
It seems that I cannot control the y-axis of a single window. My idea is to add configuration in `paneOptions`.

**Reply by mawsyh** [reply]
> It seems that I cannot control the y-axis of a single window. My idea is to add configuration in `paneOptions`.

If I have understood correctly, you said that you can not do anything with yAxis, which was what I intended to do, I was trying to replica the functionality of "bybit" in yAxis (which is disabled like most of other sites)

**Comment by mawsyh** [comment]
I can also add this feature if necessary [https://github.com/liihuu/KLineChart/discussions/148#discussion-3464654](url)
